Classes/PassiveTreeView.lua

```diff
@@ -467,10 +467,30 @@ function PassiveTreeViewClass:Draw(build, viewPort, inputEvents)
 						SetDrawColor(1, 0, 0)
 					elseif hoverNode.type == "Socket" then
 						-- Hover node is a socket, check if this node falls within its radius and color it accordingly
-						for index, data in ipairs(build.data.jewelRadius) do
-							if hoverNode.nodesInRadius[index][node.id] then
-								SetDrawColor(data.col)
-								break
+						local socket, jewel = build.itemsTab:GetSocketAndJewelForNodeID(hoverNode.id)
+						if jewel
+							and jewel.jewelRadiusIndex
+							and data.jewelRadius[jewel.jewelRadiusIndex].label == "Variable" then
```

Comment:
I'd make this into a separate bool which is aptly named. `isThreadOfHope` perhaps.